### DISCUSSION TITLE
can this method used in atmshpere?

### DISCUSSION THREAD

**Discussion by wongchieh** [Parcels-code/Parcels#1036] [Q&A]
hello!

I wonder whether this method could be used to simulate parcels' movement in atmosphere? if it can, and is there conditions for the data resolusion?

**Comment by erikvansebille** [comment]
Dear @wongchieh. I'm moving this Issue to a Discussion topic, so that others can also contribute. We haven't run atmospheric simulations ourselves, so I don't know whether it is possible. But in principle if you have 3D velocity fields (in m/s) then you should be able to integrate those fields with parcels?
